ykrt/yklua_tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+
+MODE=release
+
+if [ ! -z $PRELINK_PASSES ]; then
+  echo "yk-here"
+  yk-config ${MODE} --prelink-pipeline "${PRELINK_PASSES}" --cflags 
+else
+  yk-config ${MODE} --cflags
+fi
+
+if [ ! -z $LINKTIME_PASSES ]; then
+  yk-config ${MODE} --postlink-pipeline "${LINKTIME_PASSES}" --ldflags
+else
+  yk-config ${MODE} --ldflags
+fi
+
+make clean && make YK_BUILD_TYPE=release
+
+cd tests
+
+LUA=../src/lua
+# XXX we should repeat, say, 5 or 10 times to try to rule out non-deterministic
```

Comment:
Since there is repetition in the script that calls this one, do we need repetition here?